packages/styles/src/themes/natura/natura-dark.ts

```diff
@@ -7,15 +7,15 @@ export const naturaDark: ITheme = {
   dark: false,
   roundness: 4,
   colors: {
-    primary: natura.primary.colorBrdNatPrimaryYellow,
-    secondary: natura.primary.colorBrdNatPrimaryOrange,
+    primary: natura.primary.colorBrdNatPrimaryBrown,
+    secondary: natura.primary.colorBrdNatPrimaryYellow,
     accent: natura.primary.colorBrdNatPrimaryOrange,
-    background: "#FAFAFA",
-    surface: natura.primary.colorBrdNatPrimaryWhite,
+    background: "#222222",
```

Comment:
Doesn't this color come from the tokens?
I guess it would be great practice to either indicate this with a comment or embed it in the tokens if it belongs there.

Reply:
maybe we can create an "Other" category or something and not show it in Docs